exercises/pangram/canonical-data.json

```diff
@@ -1,97 +1,90 @@
+  "version": "2.0.0",
   "cases": [
     {
-      "description": "Check if the given string is a pangram",
-      "comments": [
-        "Output should be a boolean denoting if the string is a pangram or not."
-      ],
-      "cases": [
-        {
-          "description": "sentence empty",
-          "property": "isPangram",
-          "input": {
-            "sentence": ""
-          },
-          "expected": false
-        },
-        {
-          "description": "recognizes a perfect lower case pangram",
-          "property": "isPangram",
-          "input": {
-            "sentence": "abcdefghijklmnopqrstuvwxyz"
-          },
-          "expected": true
-        },
-        {
-          "description": "pangram with only lower case",
-          "property": "isPangram",
-          "input": {
-            "sentence": "the quick brown fox jumps over the lazy dog"
-          },
-          "expected": true
-        },
-        {
-          "description": "missing character 'x'",
-          "property": "isPangram",
-          "input": {
-            "sentence": "a quick movement of the enemy will jeopardize five gunboats"
-          },
-          "expected": false
-        },
-        {
-          "description": "another missing character, e.g. 'h'",
-          "property": "isPangram",
-          "input": {
-            "sentence": "five boxing wizards jump quickly at it"
-          },
-          "expected": false
-        },
-        {
-          "description": "pangram with underscores",
-          "property": "isPangram",
-          "input": {
-            "sentence": "the_quick_brown_fox_jumps_over_the_lazy_dog"
-          },
-          "expected": true
-        },
-        {
-          "description": "pangram with numbers",
-          "property": "isPangram",
-          "input": {
-            "sentence": "the 1 quick brown fox jumps over the 2 lazy dogs"
-          },
-          "expected": true
-        },
-        {
-          "description": "missing letters replaced by numbers",
-          "property": "isPangram",
-          "input": {
-            "sentence": "7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog"
-          },
-          "expected": false
-        },
-        {
-          "description": "pangram with mixed case and punctuation",
-          "property": "isPangram",
-          "input": {
-            "sentence": "\"Five quacking Zephyrs jolt my wax bed.\""
-          },
-          "expected": true
-        },
-        {
-          "description": "upper and lower case versions of the same character should not be counted separately",
-          "property": "isPangram",
-          "input": {
-            "sentence": "the quick brown fox jumps over with lazy FX"
-          },
-          "expected": false
-        }
-      ]
+      "description": "empty sentence",
+      "property": "isPangram",
+      "input": {
+        "sentence": ""
+      },
+      "expected": false
+    },
+    {
+      "description": "perfect lower case",
```

Comment:
I was wondering if we should maybe rephrase this test case or add something to the README to describe what a perfect pangram is.

Reply:
That sounds like a good idea. (different PR though for the README.  `"perfect pangram in lower case"` might be a better description of the test

Reply:
Context being the pangram, which is why I think that the wording ended up being the way it became.

Reply:
I am in favour of adding to the readme as wel.

Reply:
@kotp @SleeplessByte If we do a separate PR for the readme, is this PR ok to be merged?

Reply:
Yes, I think so.